shared/src/main/java/com/couchbase/lite/Result.java

```diff
@@ -75,30 +80,25 @@ public long getSequence() {
      * @return the document object.
      */
     public Document getDocument() {
```

Comment:
`getDocument()`, `getDocumentID()` and `getSequence()` will be kept as a package level method.